config/routes.rb

```diff
@@ -38,6 +38,7 @@
             end
           end
         end
+        resources :docker_registries, :except => [:new, :edit]
```

Comment:
Originally, I did registries but I couldn't get the action permission tests to pass. If you know how to do that, then let me know.